api-reference/docstubs/api/user-delta.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "delta"
+description: ""
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# delta
+
+Namespace: microsoft.graph
+
+
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Determine scopes **|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|**TODO: Determine AppOnly scopes **|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /users/delta
+GET /me/managedDevices/{managedDeviceId}/users/delta
+```
+
```

Comment:
In the live topic, we manually created a section called "Query parameters" to include the $deltatoken and $skiptoken, and another sub-section of the OData query parameters. That follows the ongoing [parameters guidance](https://msgo.azurewebsites.net/add/document/guidelines/parameters.html).

What do we intend to do in the future - have authors manually add this, or have partners include such in the CSDL?

Reply:
Angela - I read the guidance and the examples, and I'm not convinced by it.  Both deltatoken and skiptoken are defined by OData, but more than that, they are really opaque elements that developer should not need to interpret or understand, for a request.  So I don't think this is a good example for parameters guidance.

Some of this will be in CSDL and could be auto-generated, but assumes workloads actually add this information.